### DISCUSSION TITLE
Welcome to PolyGlot Discussions!

### DISCUSSION THREAD

**Discussion by DraqueT** [DraqueT/PolyGlot#971] [General]
Bloop. 

We can talk here. If you like. 

Not much else to say. Hello. :3

**Comment by lrruiz** [comment]
Hello, and thank you for your efforts! I'm glad to see you're still active. This may have a high probability of already being asked in one of the previous 970 posts, but I want to know: is the totality of Lodenkur going to be shown as an example, or do parts of it contain very personal thoughts?

**Reply by DraqueT** [reply]
Heyo, thanks the interest! Anyhow, Lodenkur is something that I need to get back to at some point. Right now, the totality of it is shared, and any updates that I make to it will be shared as well. It's ultimately part of a much larger body of creative work that's interconnected, but kind of on the back burner at the moment due to other projects that I have going.

If you're interested in the culture that surrounds the language, I maintained a story ask blog for a few years dedicate to it. ^^
https://sulmere.tumblr.com/

**Comment by Carter-Usul** [comment]
Hi, I just discovered polyglot and i'm really excited about it, i've looked for something like this before and could not find anything good so thank you so much for putting this together. I'm new to regex and programming language in general so i'm not really sure how to go about it but I'm trying to set up my declensions to add a different suffix to a word depending on if the word ends in a vowel. So dama becomes damaj and druv should become druvaj. I figured out the formula to add a specific suffix if it ends in a vowel using ^.+([aiou])$ but is there a more efficient code to ensure it adds aj after a consonant that doesn't require the prior formula with all my consonants replacing aiou? It seems so long and weighty
